settings-viewEmail.adoc

```diff
@@ -83,7 +83,6 @@ Ask me:: As soon as you open an email where a sender has requested a read receip
 ** Click image:graphics/close.svg[close icon, width=20] to decline to send a read receipt. 
 ** Click *Send read receipt* to let the sender know that you have read the message.
 
-Ask Me:: Choose this option to receive a confirmation before sending a read receipt.

```

Comment:
This option is present in the dropdown. Any reason to delete it?

Reply:
I misunderstood.  Fixed.